css/responsive/tablet.styl

```diff
@@ -2,10 +2,11 @@ $cols = 12
 $gutter = 28px
 
 .container
-  width: 750px
+  width: 90%
```

Comment:
I dont think we should go fluid until the mobile/one column layout. it just makes more work and I don't think there is an advantage